docs/theming/developers.md

```diff
@@ -21,42 +22,113 @@ order: 4
   import '@rhds/elements/rh-blockquote/rh-blockquote.js';
   import '@rhds/elements/rh-button/rh-button.js';
   import '@rhds/elements/rh-card/rh-card.js';
+  import '@rhds/elements/rh-code-block/rh-code-block.js';
   import '@rhds/elements/rh-cta/rh-cta.js';
   import '@rhds/elements/rh-pagination/rh-pagination.js';
   import '@rhds/elements/rh-tabs/rh-tabs.js';
   import '@rhds/elements/rh-tag/rh-tag.js';
   import '@rhds/elements/rh-tile/rh-tile.js';
 </script>
 
-## Context Protocol
+## Context protocol
 
-## Style Queries
+The context protocol is an [open protocol](https://github.com/webcomponents-cg/community-protocols/blob/main/proposals/context.md) developed by the web components community to support the passing of data between components.
+
+Our system utilizes this protocol with the setting of the `color-palette` attribute on a provider element which passes it's context data of `light` or `dark` on to it's children.  By doing so we can ensure accessible colors are applied given any possible change in context value higher up in the DOM tree. 
+
+This is important not only for helping us ensure we maintain our own design guidelines, but also for accessibility compliance and enabling great experiences for all our users.
+
+### Current implementation
+
+The context protocol is enabled by two [ReactiveControllers](https://lit.dev/docs/composition/controllers/): the [provider controller](https://github.com/RedHat-UX/red-hat-design-system/blob/main/lib/context/color/provider.ts), and the [consumer controller](https://github.com/RedHat-UX/red-hat-design-system/blob/main/lib/context/color/consumer.ts).  These controllers work in tandem to provide the context data, and or consume it as a child element.  Custom elements can implement only the consumer, provider, or both depending on the needs of the particular element.
+
+#### Providers
+
+Custom Elements that implement the provider controller are elements that provide their own context and can override that of their parent.  Elements such as `<rh-surface>`, `<rh-card>`, and `<rh-accordion>` are examples of such context providers.  If a provider contains a set `color-palette` attribute, it will override any parent set context, and pass it's context on to its children.
```

Comment:
```suggestion
Custom Elements that implement the provider controller are elements that provide their own context and can override that of their parent. Elements such as `<rh-surface>`, `<rh-card>`, and `<rh-accordion>` are examples of such context providers. If a provider contains a `color-palette` attribute, it will override any parent context, and pass its context on to its children.
```